src/libraries/Microsoft.PowerFx.Core/Public/Values/CollectionTableValue.cs

```diff
@@ -35,6 +36,7 @@ internal CollectionTableValue(IRContext irContext, IEnumerable<T> source)
 
             _sourceIndex = source as IReadOnlyList<T>;
             _sourceCount = source as IReadOnlyCollection<T>;
+            _sourceList = source as List<T>;
```

Comment:
>List [](http://example.com/codeflow?start=36&length=4)

IList<T>

Reply:
ICollection <T>

Reply:
This should now be:
    _sourceList = source as ICollection<T>;

Reply:
fyi